packages/gatsby-theme-carbon/src/components/Code/Code.js

```diff
@@ -33,7 +33,16 @@ const Code = ({ children, className: classNameProp = '', metaData }) => {
     if (metaData) {
       const metaDataObject = metaData.split(' ').reduce((obj, item) => {
         const [key, value] = item.split('=');
-        obj[key] = value;
+
+        // Checking for boolean values coming as string or not coming like in showAll prop
```

Comment:
I believe this entire logic should be offloaded to the package. I will see if I can get that working but not in this PR.